docs/product/size-analysis/uploading-builds/gradle.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Gradle Uploads
+sidebar_order: 20
+description: Configure Size Analysis uploads for Android projects using the Sentry Gradle plugin.
+---
+
+This assumes you already use the Sentry Android Gradle plugin. For full installation instructions, see the [Android Gradle configuration guide](https://docs.sentry.io/platforms/android/configuration/gradle/).
+
+## Apply the plugin
+
+```kotlin
+plugins {
+  id("io.sentry.android.gradle") version "6.0.0-alpha.4"
+}
+
+sentry {
+  sizeAnalysis {
+    enabled = providers.environmentVariable("GITHUB_ACTIONS").isPresent
+  }
+}
+```
+
+```groovy
+plugins {
+  id "io.sentry.android.gradle" version "6.0.0-alpha.4"
+}
+
+sentry {
+  sizeAnalysis {
+    enabled = providers.environmentVariable("GITHUB_ACTIONS").present
+  }
+}
+```
+
+## Set your auth token in GitHub Actions
+
+```yaml
+env:
+  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+```
+
```

Comment:
If we assume it is already installed then we don't need this. It is also Github ACtions specific.
```suggestion
```